asm/non_matchings/ovl2_3/func_800FA92C_ovl2.s

```diff
@@ -148,7 +148,7 @@ glabel func_800FA92C_ovl2
 /* 0835C4 800FAB54 0C00CAC8 */  jal   sqrtf
 /* 0835C8 800FAB58 46049300 */   add.s $f12, $f18, $f4
 /* 0835CC 800FAB5C 46000306 */  mov.s $f12, $f0
-/* 0835D0 800FAB60 0C0061C3 */  jal   func_8001870C_ovl2
+/* 0835D0 800FAB60 0C0061C3 */  jal   atan2f_ovl2
```

Comment:
For labels like this, I have a script in tools/massRename.py that you can use to remove the `_ovl` suffix on any function that's confirmed to not have a definition in another overlay

The only files I've touched that wouldn't benefit as much from this script are in ovl3 and ovl11; I am sure anything in ovl1 will be a completely unique function

Reply:
Thanks for the heads up, fixed in the next commit.